db/migrate/20170828133538_create_timetracks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTimetracks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :timetracks do |t|
+      t.string  :name
```

Comment:
Alle Felder, welche du auf `validates presence: true` setzt, sollten auch in der DB auf `NOT NULL` gesetzt werden. Dies als zusätzliche Sicherheitsmassnahme. 😃 

Einfach `t.string  :name, null: false` setzen.